salt-master.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:stretch
+FROM ubuntu:16.04
```

Comment:
PPAs were working on stretch as well - they are getting added and formulas installed bellow.
Even when you are changing the base image, you are keeping upstream saltstack repo for debian.

I would suggest keeping using stretch as is.

Reply:
I agree, keep stretch for simplicity.